Bottom: 6d2bed16e648c2894dfa70fe2a45f216b568e09d
Top:    ec2ab85affbca8af09a3b230afcf812b60f7fb85
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-04 12:07:48 +0800

vhost: enable packed virtqueues

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index d071daf..0f80b62 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -264,7 +264,8 @@ enum {
 			 (1ULL << VIRTIO_RING_F_EVENT_IDX) |
 			 (1ULL << VHOST_F_LOG_ALL) |
 			 (1ULL << VIRTIO_F_ANY_LAYOUT) |
-			 (1ULL << VIRTIO_F_VERSION_1)
+			 (1ULL << VIRTIO_F_VERSION_1) |
+			 (1ULL << VIRTIO_F_RING_PACKED)
 };
 
 static inline bool vhost_has_feature(struct vhost_virtqueue *vq, int bit)
